src/datascience-ui/data-explorer/reactSlickGrid.css

```diff
@@ -76,21 +70,31 @@
 }
 
 .react-grid-header-cell:hover {
-    background-color: var(--vscode-editor-inactiveSelectionBackground);
+    background-color: var(--override-selection-background, var(--vscode-list-hoverBackground));
 }
 
 .react-grid-header-cell > .slick-sort-indicator-asc::before {
     background: none;
-    font: normal normal normal 16px/1 codicon;
+    font: normal normal normal 10px/1 codicon;
     content: '\eaa1'; /* VS Code arrow-up codicon */
     align-items: center;
+    text-align: right;
+    display: flex;
+    padding: 3px;
+    color: var(--vscode-settings-textInputForeground);
```

Comment:
Bit hard to track through all the CSS, but this color looks a bit suspicious to me. It's from the GUI settings editor specifically.
https://code.visualstudio.com/api/references/theme-color#settings-editor-colors
I would expect it only to be used alongside textInputBackground which I'm not currently seeing. Using it against other background could lead to accessibility issues.

Reply:
Whoops, looks like the background is added later in this PR. This should be ok then as long at this icon is showing up under .slice-data. Is that the case?

Reply:
Hm this is actually used against --vscode-menu-background and it's the token used in the figma mockups

Reply:
Switching to menu-foreground after discussing with Jill. Eventually I think we want to use whatever the new table widget is using, but I'll wait till the design for that has stabilized.

Reply:
Awesome. Thanks for following up with Jill. These color usage mismatches used to cause lots of issue back in Visual Studio so I try to push being strict on them as much as possible.

Reply:
I did another pass and found a few more token mismatches for foreground vs background, so fixed those too :)